src/main/org/pantsbuild/jarjar/MainProcessor.java

```diff
@@ -52,7 +54,7 @@ public MainProcessor(List<PatternElement> patterns, boolean verbose, boolean ski
         if (kp != null)
             processors.add(kp);
         processors.add(new ZapProcessor(zapList));
-        processors.add(new JarTransformerChain(new RemappingClassTransformer[]{ new RemappingClassTransformer(pr) }));
```

Comment:
This is actually a little tricky because if we want to keep the entries but not rename them, JarTransformer still needs to know about that somehow to skip over them.